terraform-iac/modules/app/app.tf

```diff
@@ -205,7 +202,7 @@ EOF
 
 module "postman_test_lambda" {
   source                        = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v2.3.0"
```

Comment:
I see you didn't update the Postman test lambda. That's probably a good thing, because @mzroth and I are looking into a bug with it right now.

Reply:
I was checking that out too.

Reply:
thanks for catching that. I updated it now

Reply:
Once we get a fix released, we'll likely need to go to `v3.0.2`.

Reply:
fix for what?

Reply:
for the postman lambda. I'm working on the fix now.

Reply:
@yoshutch - I'll drop an issue over in `hw-fargate-api` with more detail about what's broken. One sec.

Reply:
https://github.com/byu-oit/hw-fargate-api/issues/242